File: web3rpc/rpc-specs/code-samples/python/eth/filter/getFilterLogs.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

quantity = "0x11b5f0ef802c98569efa8459cf063ed5"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_filter_logs(quantity)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/filter/getFilterLogs.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const id = '0xca6c12a3ecd1b44bb77f7b6536b7ce65'
    
    provider.eth.getFilterLogs(id, {}, (err, data, response) => {
        console.log(data);
    });
}
)()